### DISCUSSION TITLE
How to install solidus on debian 12 ?

### DISCUSSION THREAD

**Discussion by alainbealiah** [solidusio/solidus#5664] [Troubleshooting]
Hi ! is there a working process to install solidus on debian 12 ?
I tried this process :

```bash
sudo apt update
sudo apt install build-essential libssl-dev zlib1g-dev libreadline-dev libsqlite3-dev
gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
curl -sSL https://get.rvm.io/ | bash -s stable
source ~/.rvm/scripts/rvm
rvm install ruby 3.1.1
rvm use ruby 3.1.1
gem install rails
rails new -T amazing_store
cd amazing_store
bundle add solidus
```

but I got an dependency error:

```
Bundler could not find compatible versions for gem "actionmailer":
  In snapshot (Gemfile.lock):
    actionmailer (= 7.1.3)  In Gemfile:
    rails (~> 7.1.3) was resolved to 7.1.3, which depends on
      actionmailer (= 7.1.3)    solidus was resolved to 2.5.0.beta1, which depends on
      solidus_core (= 2.5.0.beta1) was resolved to 2.5.0.beta1, which depends on
        actionmailer (~> 5.1.0)Does anyone succeed in this process ?
```

I tried other ways too:

Installing ruby with ```apt install ruby```, but when installing gems I got a "no permission to write in this dir"
Installing default ruby version (below 3.1) with rvm but it result in a compilation error

I'm out of idea and does not understand why things are not working, I'm using latest stable version in everything

**Comment by jarednorman** [comment]
Try again with Rails 7.0 or using Solidus's main branch instead of the version on Rubygems. I don't think we've cut the release with Rails 7.1 support yet.
